### PR TITLE
Features/attributes parlence best

### PR DESCRIPTION
**Pull request by dpopleton**

### CODE REVIEW

File: src/ensembl/production/metadata/api/models/genome.py [hunk 1]
```diff
@@ -26,6 +26,7 @@ class Genome(Base):
     assembly_id = Column(ForeignKey("assembly.assembly_id"), nullable=False, index=True)
     organism_id = Column(ForeignKey("organism.organism_id"), nullable=False, index=True)
     created = Column(DATETIME(fsp=6), nullable=False)
+    is_best = Column(TINYINT(1), nullable=False)
```
Comment (marcoooo) [inline src/ensembl/production/metadata/api/models/genome.py:29]: if not `nullable`, I believe it should have a default to `0`, right?

File: src/ensembl/production/metadata/api/sample/ensembl_metadata/table.sql [hunk 2]
```diff
@@ -176,7 +176,9 @@ CREATE TABLE genome
     genome_uuid varchar(128) not null,
     assembly_id int          not null,
     organism_id int          not null,
-    created     datetime(6) not null,
+    created     datetime(6)  not null,
+    is_best     tinyint(1)   not null,
```
Comment (marcoooo) [inline src/ensembl/production/metadata/api/sample/ensembl_metadata/table.sql:180]: Would then add default to `0`

File: src/ensembl/production/metadata/updater/core.py [hunk 3]
```diff
@@ -91,40 +91,49 @@ def process_species(self, species_id, metadata_uri, taxonomy_uri, db_uri):
             genebuild_release_status = conn.check_release_status(self.genebuild_dataset.dataset_uuid)
 
             if organism_status == "New":
-                print ("New organism")
+                print("New organism")
```
Comment (marcoooo) [inline src/ensembl/production/metadata/updater/core.py:94]: Are those `print` statement something for logging? If so, please use a logger. * which in the end should be essential part of figuring out what could go wrong in a loading.

File: src/ensembl/production/metadata/updater/core.py [hunk 4]
```diff
@@ -342,7 +352,7 @@ def get_assembly_sequences(self, species_id, assembly):
                         name = preferred_name
 
                 # Combine all unique names with ";". If a name appears in multiple sequences with the same accession,
-        #        name = ";".join(info["names"])
+                #        name = ";".join(info["names"])
```
Comment (marcoooo) [inline src/ensembl/production/metadata/updater/core.py:355]: If this comment is not meant to be retired soon, please simply delete the line, it's confusing for the future developer who might come after you.

**Review by marcoooo — CHANGES_REQUESTED** [review]

File: src/ensembl/production/metadata/api/models/genome.py (hunk 1 above)
Comment (dpopleton) [reply src/ensembl/production/metadata/api/models/genome.py:29]: That is a good point that extends beyond is_best. i have added it to all the not null booleans.

File: src/ensembl/production/metadata/api/sample/ensembl_metadata/table.sql (hunk 2 above)
Comment (marcoooo) [inline src/ensembl/production/metadata/api/sample/ensembl_metadata/table.sql:180]: This should reflect the default then, right? 
```suggestion
    is_best     tinyint(1)   not null default 0,
```

**Pull request merged**